packages/core/src/components/plekinfo-card/plekinfo-card.scss

```diff
@@ -0,0 +1,143 @@
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/variables/media-query-breakpoints";
+@use "~dso-toolkit/src/variables/units";
+@use "~dso-toolkit/src/components/plekinfo-card";
+
+@use "~dso-toolkit/src/components/insert/insert";
+@use "~dso-toolkit/src/components/delete/delete";
+
+@include utilities.box-sizing();
+
+:host {
+  display: block;
+
+  .dso-plekinfo-card-container {
```

Comment:
Het is niet nodig om selectors in de `:host` selector te plaatsen als ze niets van `:host()` nodig hebben

Reply:
Gedaan